spinalcordtoolbox/scripts/sct_dmri_separate_b0_and_dwi.py

```diff
@@ -200,7 +200,13 @@ def main(args=None):
     # Average b=0 images
     if average:
         printv('\nAverage b=0...', verbose)
-        run_proc(['sct_maths', '-i', b0_name + ext, '-o', b0_mean_name + ext, '-mean', 't'], verbose)
+        img = Image(b0_name + ext)
+        out = img.copy()
+        dim_idx = 3 # 3 dimensions + time
```

Comment:
Nit: I'm not sure we need to be this explicit about what `3` means here... I think it might be clear from context.

Reply:
There's also an identical line for `dwi` images lower down.